plugins/shared/info-panel/index.js

```diff
@@ -207,10 +207,13 @@ class InfoPanel {
 
                 // Wire up the expand/collapse trigger
                 let $trigger = $error.find(".tota11y-info-error-trigger");
+                let $desc = $error.find(".tota11y-info-error-description");
+
                 $trigger.click((e) => {
                     e.preventDefault();
                     e.stopPropagation();
                     $trigger.toggleClass(COLLAPSED_CLASS_NAME);
```

Comment:
Does having the classname on `$trigger` do anything anymore? Looks like we don't have styles for it.